addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -299,9 +320,11 @@ export class Rtc extends Record {
 
         browser.addEventListener("pagehide", () => {
             if (this.state.channel) {
-                const data = JSON.stringify({
-                    params: { channel_id: this.state.channel.id },
+                navigator.serviceWorker.controller?.postMessage({
+                    name: SW_MESSAGE_TYPE.UNEXPECTED_CALL_TERMINATION,
+                    channelId: this.state.channel.id,
```

Comment:
I'm afraid this part is subject to race conditions. e.g.
- message sent to the worker, call recovered on another tab.
- `/mail/rtc/channel/leave_call` received, call that was just started is closed?
Is it already handled? Should we sent the `rtc_session_id` in this case to prevent closing the wrong call?